File: example/fltk/hello_world.cpp
// Copyright (c) 2016, Herman Bergwerf. All rights reserved.
// Use of this source code is governed by a MIT-style license
// that can be found in the LICENSE file.

#include <FL/Fl.H>
#include <FL/Fl_Window.H>
#include <FL/Fl_Box.H>

int main(int argc, char **argv) {
  Fl::scheme("gleam");
  Fl_Window *window = new Fl_Window(300, 180, "FLTK");
  Fl_Box *box = new Fl_Box(20, 40, 260, 100, "Hello, World!");
  box -> box(FL_UP_BOX);
  box -> labelsize(36);
  box -> labelfont(FL_BOLD + FL_ITALIC);
  box -> labeltype(FL_SHADOW_LABEL);
  window -> end();
  window -> show(argc, argv);
  return Fl::run();
}

File: test/classic/hello_world.dart
// Copyright (c) 2016, Herman Bergwerf. All rights reserved.
// Use of this source code is governed by a MIT-style license
// that can be found in the LICENSE file.

import 'package:color/color.dart';
import 'package:fltk/fltk.dart' as fl;

int main() {
  fl.scheme = 'gleam';
  var window = new fl.Window(350, 180, 'FLTK');
  var box = new fl.Box(20, 40, 310, 100, 'Hello, World!');
  box.box = fl.UP_BOX;
  box.labelsize = 36;
  box.labelfont = fl.BOLD + fl.ITALIC;
  box.labeltype = fl.SHADOW_LABEL;
  box.labelcolor = fl.YELLOW;
  box.color = fl.toColor(new HexColor('#ff0000'));
  window.end();
  window.show();

  return fl.run();
}
